src/goto-programs/goto_program.cpp

```diff
@@ -14,6 +14,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <ostream>
 #include <iomanip>
 
+#include <goto-programs/remove_returns.h>
```

Comment:
I'd prefer `#include "remove_returns.h"` and placing that at the bottom of the list of includes, for consistency with how that's done in other files.

And a nit-pick: s/differnet/different/ in the commit message.

Reply:
Fixed.